setup.py

```diff
@@ -17,7 +18,8 @@
 
 with open("README.md", "r") as fh:
     long_description = fh.read()
-install_requires = ['requests', 'click', 'jsonschema', 'PyYAML', "Jinja2", "loompy"]
+
+install_requires = ['requests', 'click', 'jsonschema', 'PyYAML', "Jinja2", "loompy", "numba"]
```

Comment:
We also need `ga4gh-testbed-lib` here otherwise we get `ModuleNotFoundError: No module named 'ga4gh'` after `python setup.py install`